liqd_product/assets/scss/style.scss

```diff
@@ -58,3 +57,6 @@
 
 @import '~a4-meinberlin/meinberlin/assets/scss/utility';
 @import '~a4-meinberlin/meinberlin/assets/scss/shame';
+
+@import "layout";
+@import "components/hero-unit";
```

Comment:
The import order is important. These should be close to other layout/component imports.